tests/common/util.rs

```diff
@@ -261,7 +261,10 @@ impl AtPath {
     }
 
     pub fn write(&self, name: &str, contents: &str) {
-        let mut f = self.open(name);
+        let mut f = OpenOptions::new()
```

Comment:
This is missing a `log_info!()` statement.  Also, you can just use `File::create()` here instead.

Reply:
I used touch to create and write to write.
`at.touch(<file>);
at.write(<file>, <content>)`
However, this resulted in an empty file.